recipes/compss/activate.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+  
+export PYTHIA8="${CONDA_PREFIX}"
+export PATH=$PATH:${CONDA_PREFIX}/lib/python3.7/site-packages/pycompss/COMPSs/Runtime/scripts/user:${CONDA_PREFIX}/lib/python3.7/site-packages/pycompss/COMPSs/Runtime/scripts/utils:${CONDA_PREFIX}/lib/python3.7/site-packages/pycompss/COMPSs/Bindings/c/bin:$PATH
```

Comment:
What about symlinking the scripts to `${CONDA_PREFIX}/bin` instead? We don't usually modify PATH like this at all.